app/src/main/java/it/quip/android/adapter/NotificationHandler.java

```diff
@@ -0,0 +1,20 @@
+package it.quip.android.adapter;
```

Comment:
We havent agreed to that - interface vs interfaces and, I am fine with taht
either way.

On Sun, Oct 18, 2015 at 7:27 PM, Hasham Ali <notifications@github.com>
wrote:

> In app/src/main/java/it/quip/android/adapter/NotificationHandler.java
> <https://github.com/quipit/quipit-android/pull/7#discussion_r42332421>:
>
> > @@ -0,0 +1,20 @@
> > +package it.quip.android.adapter;
>
> lets move this to a new package called "interfaces"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quipit/quipit-android/pull/7/files#r42332421>.
>